exercise/static/exercise/chapter.js

```diff
@@ -506,6 +492,22 @@
 			return [exercise, valid, formData];
 		},
 
+		quizSubmission: function(message) {
+			var exercise = this;
+			if (message == "submit") {
+				exercise.element.find(exercise.settings.quiz_overlay_selector).removeClass("hide");
+				exercise.element.find(exercise.settings.quiz_progress_selector).removeClass("hide");
+			} else if (message == "error") {
+				exercise.element.find(exercise.settings.quiz_overlay_selector).hide();
+				exercise.element.find(exercise.settings.quiz_progress_selector).hide();
+				exercise.element.find(exercise.settings.quiz_error_selector).removeClass("hide");
```

Comment:
Is this `removeClass("hide")` same as `show()`?

Reply:
According to this, no actually: https://api.jquery.com/show/

Regarding this:

> However, could the progress bar be at the end of the exercise or both, in the start and the end? Or, could it be a sticky element? Would that work? My concern is that when the form is long, the progress bar witll be out of the view.

I added the stickiness, it actually works nicely! It was a good idea. And I think it was the best option of these, the duplicate bars looked funny I think. I added a mention of moving the sticky-definition to a helper-file. For future reference, would the `sass/helpers` be the correct place for something like this? 

Everything has also now been rebased and translations checked as well. I also run the tests now, and seems that selenium-tests now also all pass, after Veera's fix :)

Reply:
This is probably not repeatable, as `hide` sets `display=none` and `removeClass("hide")` doesn't work for that, only for the class added in the template. `show()` checks that if the element has effective `display==none`, then it will set element to `display=<default display for the element>`, thus forcing it to be visible.